DropInDemo/Features/Drop In/BraintreeDemoDropInViewController.m

```diff
@@ -261,6 +262,30 @@ - (void)tappedToShowDropIn {
         dropInRequest.payPalRequest = [[BTPayPalRequest alloc] initWithAmount:@"4.77"];
     }
 
+    if ([BraintreeDemoSettings threeDSecureRequiredVersion] == BraintreeDemoTransactionServiceThreeDSecureRequestedVersionLegacy) {
+        dropInRequest.amount = @"10.00";
+        dropInRequest.threeDSecureVerification = YES;
```

Comment:
This conflicts with the check for `[BraintreeDemoSettings threeDSecureRequiredStatus]` above, on line 240.